From 990421fae58143fb2a23f428ea395c655bb936da Mon Sep 17 00:00:00 2001
From: Hao Lan <lanhao@huawei.com>
Date: Fri, 9 Dec 2022 11:01:58 +0800
Subject: [PATCH 249/268] net: hns3: fix the HCLGE_OPC_WOL_CFG opcode id for
 wol

driver inclusion
category: bugfix
bugzilla: https://gitee.com/src-openeuler/kernel/issues/I8EN49
CVE: NA

----------------------------------------------------------------------

Fix the HCLGE_OPC_WOL_CFG opcode id for wol.

Fixes: c3c5f044b7dc ("net: hns3: support wake on lan configuration and query")
Signed-off-by: Hao Lan <lanhao@huawei.com>
Signed-off-by: Jiantao Xiao <xiaojiantao1@h-partners.com>
Signed-off-by: Xiaodong Li <lixiaodong67@huawei.com>
---
 .../net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h b/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
index 1e5448ee4744..8477e9f1a3d7 100644
--- a/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3_common/hclge_comm_cmd.h
@@ -296,8 +296,8 @@ enum hclge_opcode_type {
 	HCLGE_PPP_CMD1_INT_CMD		= 0x2101,
 	HCLGE_PPP_MAC_VLAN_IDX_RD	= 0x2104,
 	HCLGE_MAC_ETHERTYPE_IDX_RD      = 0x2105,
-	HCLGE_OPC_WOL_CFG               = 0x2200,
-	HCLGE_OPC_WOL_GET_SUPPORTED_MODE        = 0x2201,
+	HCLGE_OPC_WOL_GET_SUPPORTED_MODE	= 0x2201,
+	HCLGE_OPC_WOL_CFG		= 0x2202,
 	HCLGE_NCSI_INT_EN		= 0x2401,
 
 	/* PHY command */
-- 
2.34.1

